### ISSUE TITLE
Collection of resources / inspiration

### ISSUE THREAD

**Issue by samuelorion** [osumontreal/OLS-2#14]
I have my own docs with just lists of resources, with a short explainer of what it is. 

I think it would be good to have a similar list on OSUM. 

Are there other nice places which list loads of useful resources. Any thoughts...?

**Comment by anproulx** [comment]
Found this great repo with many open science ressources available

https://github.com/asoplata/open-science-resources

**Comment by samuelorion** [comment]
👍 C'est malade... ! Bcp!! Merci. Ce serait bien d'intégrer ces informations!

**Comment by SamGuay** [comment]
That would be a really goood idea! 

One caveat about building our own *mini database* is that there are already many databases being built at the moment. We may want to think about what we want to select, whether we want to select resources more geared towards trainees or to everyone, do we want a static infinite document or a dynamic/book. Also, what kind of info we want associated with the resources? Do we want a paragraph that summarises what it is, who would benefit from it, in which field, the topics, etc. Also, we need to keep in mind that resources may get outdated with broken links, etc. so it may take some time to maintain it (or we don't care/people will open an issue or fix it when they see fit).

For now, since we seem to be keen on markdown files (which is nice because it is easy to transpose it on the website eventually), we could make such document in a bookdown-like way like this one that I've created as an example: https://hackmd.io/@openscience/book. It is linked to Github and can sync bothways, so people can edit it directly on HackMD, and the changes are gonna get backed up in the [resources](https://github.com/osumontreal/resources) repo. Instead of going after several hundreds of resources, we may want to target the most promising ones?

[FORRT's databse](https://forrt.org/resources) is a great example of careful reviewing resources before input them in a database. FORRT stands for Framework for Open and Reproducible Research Training, and they gather several resources to help people implement/teach OS principles and practices.

PS: while writing this comment, I dug a gem out. A friend and I populated this a while ago :joy: :arrow_right: https://strugglinggs.github.io/savior/. Please, don't use code from there, apparently, it is full of bad code practices :grimacing:.

**Comment by dannycolin** [comment]
Maybe an alternative to the never ending resources list and all the caveats that come with it (e.g broken links, maintenance) would be to create small guides on specific topics. For example, the EFF.org provides security guides geared to different groups of people (see: https://ssd.eff.org/en/module-categories/security-scenarios). I think we could adapt that idea and create a few guides like "Open Science: The Basics", "Open Science for Social Scientist" and so on.

**Comment by SamGuay** [comment]
That is a clever approach. I like their [Academic researcher's guide](https://ssd.eff.org/en/playlist/academic-researcher). I guess if we find a couple of highly interesting tools, we can always write a blog post about such tools. 

Question: How do we decide which tools we include in the *guides*?

**Comment by samuelorion** [comment]
I think I agree with many of the points here. We don't want to make just another list. And we don't want a bunch of dead links. 

But I guess we need to find a balance between having things for people to discover, vs taking the time to 'write guides', which is extremely time consuming. 

Perhaps we could have a couple of sub-sections (as in the book), and have some pieces which we find are a great start for each category. Perhaps we "develop guides" as an actual activity of OSUM!? i.e. - a kind-of journal club where a group review a blog  / site, whatever, and populate a hackMD to make a summary of the resource, in french!? [this could be an acitvity we do at our first OSUM meeting #OSUMideas hehe 

One thing that that I find myself  thinking about this issue, is, how does this fit in with 
1. What we are trying to do?
2. Is this what 'our users' want?

So, what 1. raises for me, is ... do we actually know what 'the community' wants or needs, or will engage with!? 

Perhaps  2, is something we should clarify as to help focus our efforts within the milestones outlined [here](https://github.com/osumontreal/OLS-2/blob/master/ROADMAP.md#milestones-for-ols-2) 

>Milestones for OLS-2
>- A comprehensive contribution guidelines
>- A revamped website that descibres who we are, what open science is, and why we need to do science in the open
>- MOOC for Edulib : "Introduction à la science des données sociales"

**Comment by dannycolin** [comment]
> But I guess we need to find a balance between having things for people to discover, vs taking the time to 'write guides', which is extremely time consuming.

Indeed. A good balance between those two things could be to create a curated list of max 10-15 links that we keep up to date and that contain a short descriptive for every resources. It should also contain resources targetings newcomers.

> One thing that that I find myself thinking about this issue, is, how does this fit in with
>
>    What we are trying to do?
>    Is this what 'our users' want?

Those are extremely good questions. For 1, I think our objective with the resources list is to give a way for people to explore by themselves a bit more on the basics of Open Science but with a bit of help from us by curating the list and a tl;dr. Question 2 is a bit harder to answer. We currently don't have a userbase but we can infer that lots of people at UMontreal don't know about Open Science and that they'll benefit from this curated list. However, I think we should keep in mind this question for future iteration of the list and use the data we'll get from the users who'll use that first version.
